extensions/amp-story/1.0/amp-story.js

```diff
@@ -1064,10 +1067,51 @@ export class AmpStory extends AMP.BaseElement {
       });
     }
 
+    this.buildScreenReaderBackButton_();
+
     // Will resolve when all pages are built.
     return storyLayoutPromise;
   }
 
+  /**
+   * Creates a focusable button for screen readers to navigate back.
+   * @private
+   */
+  buildScreenReaderBackButton_() {
+    const backButton = document.createElement('button');
+
+    const label = this.localizationService_.getLocalizedString(
+      LocalizedStringId.AMP_STORY_PAGINATION_BUTTON_PREVIOUS_PAGE_LABEL
+    );
+
+    backButton.classList.add('i-amphtml-story-screen-reader-back-button');
+    label && backButton.setAttribute('aria-label', label);
+    this.mutateElement(() => {
+      this.element.appendChild(backButton);
+    });
+
+    backButton.addEventListener('click', () => {
```

Comment:
I don't think we actually need this click listener? `page-advancement.js` should handle it.